Composite/Composite.Cs.Tests/Tests.cs

```diff
@@ -6,19 +6,49 @@
 
 using Composite;
 
-using SimpleComposite = Composite.DataTypes.Composite<Composite.Cs.Tests.Tests.Simple>.Composite;
+using SimpleComposite = Composite.DataTypes.Composite<Composite.Cs.Tests.Simple>.Composite;
 
-namespace Composite.Cs.Tests {
-    public class Tests {
+namespace Composite.Cs.Tests
+{
+    public class Tests
+    {
+        [Fact]
+        public void ToBatchedTest()
+        {
+            var inputSequence = Enumerable.Range(1, 10)
+                .Select(x => new Simple { Number = x, })
+                .AsLimited(6);
+            var batches = inputSequence.AsBatched(10, x => x.Number)
+                .Take(2)
+                .ToArray();
+
+            Assert.Equal(1, batches[0][0].Number);
+            Assert.Equal(2, batches[0][1].Number);
+            Assert.Equal(3, batches[0][2].Number);
+            Assert.Equal(4, batches[0][3].Number);
+            Assert.Equal(5, batches[1][0].Number);
+            Assert.Equal(6, batches[1][1].Number);
+        }
 
```

Comment:
Looks like ToBatchedTest shows that the method AsBatched does not follow the described behavior because in the second batch the weight of elements greater than the batch size.

Reply:
@v-ilin: that's right. This issue is unrelated to this PR, though. Mentioning this issue in #12 advised.

Reply:
@vkamiansky  Thanks, I copied that to #12.